src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
     * Added `SignalableCommandInterface` interface
  * Added `TableCellStyle` class to customize table cell
  * Removed `php ` prefix invocation from help messages.
+ * Added `GithubAnnotationsReporter` to render errors as annotations in a Github Action
```

Comment:
This will probably be in 5.3.0 as 5.2 is already in feature freeze since quite some time.